keyboards/keebio/bdn9/keymaps/vial/rules.mk

```diff
@@ -0,0 +1,9 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+MOUSEKEY_ENABLE = yes
+CONSOLE_ENABLE = yes
+COMMAND_ENABLE = no
+VIAL_ENABLE = yes
+VIAL_ENCODERS_ENABLE = yes
+ENCODER_ENABLE = yes
+VIAL_INSECURE = yes
```

Comment:
same as above, should be removed

Reply:
Is it the insecure part or the lines for encoders?